ilm200Sup/ilm200.proto

```diff
@@ -0,0 +1,22 @@
+OutTerminator = "\r";
+InTerminator = "\r";
+ReplyTimeout = 2000;
+
+getLevel {
+    out "@\$1R\$2";
+    in "%f";
```

Comment:
That is true. The emulator should be replying `R%f` because when it didn't the VI missed the most significant digit. Didn't spot any complaints in the logs